backend/src/api/item-type/content-types/item-type/schema.json

```diff
@@ -0,0 +1,27 @@
+{
+   "kind": "collectionType",
+   "collectionName": "item_types",
+   "info": {
+      "singularName": "item-type",
+      "pluralName": "item-types",
+      "displayName": "Item Type"
+   },
+   "options": {
+      "draftAndPublish": true
+   },
+   "pluginOptions": {},
+   "attributes": {
+      "akeneo_code": {
+         "type": "string",
+         "required": true,
+         "unique": true,
+         "regex": "[a-z][a-z0-9]*"
```

Comment:
Not sure if this is getting used, but we need to support underscores.

Reply:
Fixed.